wdl/GATKSVPipelineSingleSample.wdl

```diff
@@ -262,10 +262,10 @@ workflow GATKSVPipelineSingleSample {
     RuntimeAttr? runtime_attr_depth_merge_pre_clusterbatch
 
     # Reference panel standardized caller VCFs
-    Array[File] ref_std_manta_vcfs
-    Array[File] ref_std_wham_vcfs
-    Array[File]? ref_std_melt_vcfs
-    Array[File]? ref_std_scramble_vcfs
+    File ref_std_manta_vcfs  # list of files, one per line
```

Comment:
Do you think we should change these input names to eg `ref_std_manta_vcf_list` for consistency's sake? I'm not hung up on it but usually when I see an input name that's pluralized I assume it's going to be an array.

Reply:
Fair enough

Reply:
Updated & re-validated WDLs and JSONs with womtool & Terra validation script